src/shapes/object.class.js

```diff
@@ -596,6 +596,14 @@
       ' strokeLineCap strokeDashOffset strokeLineJoin strokeMiterLimit backgroundColor clipPath'
     ).split(' '),
 
+    /**
+     * List of properties to consider for animating colors.
+     * @type Array
+     */
+    colorProperties: (
+      'fill stroke backgroundColor'
+    ).split(' '),
```

Comment:
i want to merge this as it is, but i think i will come back and move `colorProperties` to animation util level maybe.
Too busy to think of it now and not so important.

Reply:
I see, the reason I put it on the object class is because I thought it might be easier to create a new shape class  with a custom property name that involved coloring but as you like :)

Reply:
i know but so many options.